sc/src/main/webapp/WEB-INF/tiles/views.xml

```diff
@@ -8,6 +8,7 @@
         <put-attribute name="title" value="Home &lt; "/>
         <put-attribute name="content">
             <definition template="/WEB-INF/jsp/tiles/views/home.jsp">
+                <put-attribute name="rel" value="/WEB-INF/jsp/tiles/views/home/release-stats.jsp"/>
```

Comment:
Can we give this a nicer, more explicit name, rather than "rel"?

Reply:
Oh, right! Thanks for spotting it.